### ISSUE TITLE
Monero "Policy & Regulation" Initial Meeting: September 25th, 2019, 17:00 UTC 

### ISSUE THREAD

**Issue by binaryFate** [monero-project/meta#392]
### "Policy & Regulation" Think Tank / Working Group
Over the last years and especially last months, several people in independent circles and channels around the Monero community have expressed the desire to set up a "Policy & Regulation" entity (wording mine, pardon me if I misrepresent all ideas around).

A [reddit thread](https://www.reddit.com/r/Monero/comments/d5igia/monero_regulatory_workgroup) was posted on the topic. Please read it, there are some interesting comments there.

**Practical examples of initiatives in scope**
To start discussion towards thinking what this entity could aim at, here are some items that I spew from the top of my head. These are both past/current/possible future initiatives that IMO could potentially be within scope.

* the #eu-crypto-privacy discussions that was set up about a year ago (I think?) when some high-level EU technocrats made noise about banning some crypto primitives such as ring signatures.

* the initiative of Midipoet to follow and take part in discussions of two working groups of the ITU (UN telecommunications subgroup), ["Digital Currency"](https://extranet.itu.int/sites/itu-t/focusgroups/dfc/SitePages/Home.aspx) & ["Application of Distributed Ledger Technology"](https://extranet.itu.int/sites/itu-t/focusgroups/fgdlt/SitePages/Home.aspx).

* I am personally interested to design (on paper only, I do not mean coding) a "GDPR-coin", that respects both the GDPR and the principle of decentralization, if at all possible. Then analyze how Monero fits with respect to these theoretical specifications.
Or just show directly that Monero is the most GPDR-compliant cryptocurrency, but it's less fun of an approach :)

* Get in touch with EU institutions. We have no recurring channel of communications or contacts, this is stupid. Last year I got to a panel within parliament on cashless societies, this was a useful event. But the organizers struggled to get a hold of someone in the community willing to go and it worked with some luck this time; how many missed opportunities of that type? 
I mention EU as I am mostly familiar with the institutions in Brussels, but the same is true for any other relevant policy center.

* Analyse existing reports and write complementary (contradictory?) papers. For instance a large report ["Blockchain Now and Tomorrow"](https://ec.europa.eu/jrc/en/publication/eur-scientific-and-technical-research-reports/blockchain-now-and-tomorrow) from the Joint Research Center of the European Commission was just released. I skimmed through the executive summary only and it already warrants some reactions.

* KYC (there is a growing debate that applying plain simple passport-KYC is sometimes quite detrimental -- see all the data breach and the ID documents linked).

* Blockchain analytics companies: they have a vested interest to paint the crypto world in a way that makes analysing blockchain history on top of normal KYC necessary and mandatory. This is antagonist to the idea of fungibility and Monero users should fight back this narrative.

* The new FATF recommendations, and in particular the travel rule. So many open questions on this one.

**Key meta questions** 
xmrhaelan posted this list of key initial questions:

* Do we need a separate workgroup for this or should it fall within the current Monero Outreach Workgroup?

* What directions, ideas, priorities exist among Monero that relate to policy and regulation?

* How should we organize going forward? Do we need to setup another IRC channel for communication or can we use the existing Monero Outreach channels and stick with periodic community updates?

* What other considerations should be made?

There were several poeple answering in the reddit thread. Maybe somebody can go through before the meeting and summarize the positions, and then further points and opinions be raised at the meeting?

AJS added (in private) the following important question:

* To which extent should it be Monero specific vs. cryptocurrency agnostic?

**Now let's get to an effective agenda**

So I think maybe if everyone could chime in on (i) whatever you have in mind that this entity could be interested about or try to do in practice, so we discuss in terms that are not too abstract, and (ii) general questions and items that definitely should be covered during meeting, please add to the list.
Hopefully we will get to a constructive agenda, to be finalized in time for meeting.

_Note that we will not necessarily keep having meetings in the future. Everything is up to be decided. This one is more of a meta meeting for defining everything else going forward._

On a personal note I'm super excited about things moving on this front, and I know many who are as well. Let's put FUNgibility back into policies and regulations!

----------------------------------------------
**Location**

[Freenode](https://kiwiirc.com/nextclient/#irc://irc.freenode.net/#monero-community) | [Mattermost](https://mattermost.getmonero.org/monero/channels/monero-community) | [Slack](https://monero.slk.com/) | Irc2P

Please test the relays shortly before using. If there are any issues, please use Freenode IRC directly.

Please PM [SGP on Reddit](https://www.reddit.com/message/compose/?to=SamsungGalaxyPlayer&subject=Monero%20Slack%20invite%20(from%20GitHub)&message=Hello%20please%20send%20a%20Monero%20Slack%20invite%20to%20the%20following%20email:) with your email for a Slack invite if desired.

* `#monero-community`

 **Time**

17:00 UTC
13:00 ET
12:00 (noon) CT
10:00 PT

[Use this timezone calculator to convert UTC to your time zone](https://www.timeanddate.com/worldclock/converter.html?iso=20190608T170000&p1=tz_et&p2=28&p3=111&p4=49&p5=179&p6=70&p7=224&p8=48&p9=136).

**Comment by umma08** [comment]
nice initiative. i will be present.

**Comment by arnuschky** [comment]
same

**Comment by SamsungGalaxyPlayer** [comment]
This is related to my work, so I will most likely be present.

**Comment by hooftly** [comment]
Personally I feel that although steps are being taken to help educate... education is only half the battle.  Regulators can be sent literature and people can attend conferences to hopefully get some materials in the hands of attendee's as well as attending regulators but in all honesty im am not sure this is enough.  

Thing is any industry that wants regulations to go a certain way requires a constant lobbying presence especially in the USA.  

I highly doubt banks and financial institutions are sitting idly by letting these regulatory frameworks develop.  They are most likely in the thick of it lobbying regulators and policy makers alike to push things in their desired direction.  If privacy coin and privacy advocates are not willing to go to the same lengths I fear the battle is already a losing one.

All i can find is coincenter and while they provide great resources it seems they do not actively lobby government for responsible and fair regulations.  

You can "educate" regulators all day but in the end if you don't give them a reason to listen it ultimately falls on deaf ears. 

When I was in DC attending the Annual Blockchain summit where Regulators were discussing all of this one thing was painfully obvious...  No community projects wete represented let alone talked about.  It was all Ripple and EOS and Blockchain startups.  lots of talk of STOs and ICOs but not one thing about privacy coins or possible regulation was discussed.  This needs to change for any education to be at all effective.

Anyways would live to hear thoughts maybe I'm missing something and am way off base.

**Comment by xmrhaelan** [comment]
Here are a few agenda items I’ve seen come up:
- How to find the balance between setting regional examples of engaging with regulators vs. scaling to engage with regulatory bodies internationally and among various jurisdictions? Where to prioritize and focus?
- a more effective name for this may be the “Monero Legal Workgroup”, so we do not create inadvertent barriers to engaging with the regulators we want to engage with (see intj440 comment on Reddit
- how can we best organize so as to enhance the skills of all interested parties and maximize the impact we can create together?

**Comment by xmrhaelan** [comment]
@hooftly those are good observations. Please try to turn them into suggestions.

**Comment by hooftly** [comment]
Fair Enough.

To turn the observations into possible agenda items I would summarize it as the following:
  
  -  People are needed to publicly engage regulators on an ongoing basis.  How can this be achieved?
  - A discussion about Reality Vs. Hope.  Many people that I engage with believe even if Privacy Coins were regulated or banned it wouldn't matter and people would continue to use them.  In reality, this is simply just not true in most cases.  How can this scenario be avoided and what can we do now to try and ensure the worst does not become a reality.
  - Piggybacking on the balance vs. scaling obviously this is an issue. This does need to be discussed at a regional level as that is the only way any of us will make an impact in our respective jurisdictions.  Perhaps a discussion about different Chapters of the Legal Workgroup could be established specialized per region (Tall order I know).

**Comment by umma08** [comment]
I would like to add the Privacy Series initiative (www.privacyseries.org) to the discussion, as i believe a physical "discursive" event is required. the event is proposed to be inclusive of main stakeholder groups, with the goal of achieving clarity on a number of regulatory and legislative issues.

**Comment by binaryFate** [comment]
@hooftly

I agree with having somebody "on the ground" eventually, once it becomes feasible. Maybe one day the community or some corporate/individual sponsor could support such a thing; it will be useful then to have a supporting group with supporting material, connections etc.

==============================
## Tentative agenda

* Meet and greet

* Scope of this group
  * Purpose(s) / Objective(s)
  * Categorization of initiatives that could be pursued
  * General domain of expertise (Monero-specific or general privacy or...)

* Should it be part of an existing group (outreach was mentioned)

* How to organize going forward

Any other important point?

**Comment by SamsungGalaxyPlayer** [comment]
```
[2019-09-25 12:02:00] <binaryFate> Hello everyone and welcome to this meeting
[2019-09-25 12:02:06] <ArticMine> Hi
[2019-09-25 12:02:09] <binaryFate> Agenda is here: https://github.com/monero-project/meta/issues/392
[2019-09-25 12:02:13] <xmrmatterbridge> <sgp_> Hello
[2019-09-25 12:02:17] <binaryFate> Please say hi if you're here and interested :)
[2019-09-25 12:02:17] <almutasim> Hello.
[2019-09-25 12:02:21] <midipoet> hello
[2019-09-25 12:02:27] <de_anna> hey there
[2019-09-25 12:02:37] <krongle> Hi
[2019-09-25 12:03:17] <binaryFate> Before we start and so we know how to deal with time, is there any big item that you think is missing in the agenda?
[2019-09-25 12:03:21] → arnuschky joined (~arnuschky@p57AA41D5.dip0.t-ipconnect.de)
[2019-09-25 12:04:00] <arnuschky> Hello fellow fans of regulation and governmental oversight!
[2019-09-25 12:04:00] <binaryFate> almutasim I take it xmrhaelan cannot be present?
[2019-09-25 12:04:16] <almutasim> Yes, I think I'm standing in for him.
[2019-09-25 12:04:45] <ArticMine> Yes leveraging Monero privacy for travel rule compliance
[2019-09-25 12:05:07] <binaryFate> Ok so first, scope of this potential group
[2019-09-25 12:05:22] <binaryFate> 1) Purpose, 2) Practical initiatives, 3) domain of expertise
[2019-09-25 12:05:31] <binaryFate> First purpose. What do you think we can/should aim at?
[2019-09-25 12:05:38] <krongle> How about discussing alternatives to KYC that might be more privacy friendly - like IMRA proposed by https://privacybydesign.foundation/en/
[2019-09-25 12:05:38] <midipoet> regulatory clarity
[2019-09-25 12:06:01] <midipoet> for 1)
[2019-09-25 12:06:21] <almutasim> Also, influencing the creation of new laws.
[2019-09-25 12:06:22] <de_anna> and clarity on monero in relation to regulations
[2019-09-25 12:07:05] <ArticMine> Regulatory interpretation
[2019-09-25 12:07:14] <binaryFate> For me, sadly I perceive a lot of regression in regulations and that we will need to defend non-transparent transactions. It's mostly about making sure a Monero-compatible narrative exists besides the main one currently praising transparent transactions, chain analytics as a requirement etc
[2019-09-25 12:08:22] <binaryFate> So explaining and educating (i) why Monero is not by itself antagonist to legitimate regulation and as a bonus (ii) why it is actually better than transparency by default (why fungbility)
[2019-09-25 12:08:36] <ArticMine> I agree but also I would add leveraging privacy for AML / KNC compliance taking the narrative even further
[2019-09-25 12:08:54] <de_anna> agreed for the most part, it would be beneficial to take a two-pronged approach: regulatory interpretation as it stands and progressive regulatory proposals based i) & ii) in the context of privacy and financial privacy as a fundamental right
[2019-09-25 12:08:56] <midipoet> i think ii) can be a stronger argument. perhaps fungibility + data protection
[2019-09-25 12:09:04] <arnuschky> I see this happening as a result of the practical application of KYC and AML. Ie companies trying to do their best in order not to get prosecuted, and jumping to chainalysis and co because of the lack of alternatives.
[2019-09-25 12:09:05] <krongle> I agree with the idea of influencing new laws. Although the environment is tough at the moment, it is also a moment when the industry as a whole has to decide (and negotiate) what the FATF guidelines actually mean in our context.
[2019-09-25 12:10:00] <binaryFate> Fyi, ArticMine and I and the rest of the core team lately receive questions from exchanges and journalists about pretty much "why Monero prevents FATF rules"
[2019-09-25 12:10:02] <arnuschky> I think providing some solution for companies needed to fulfill AML/KYC checks that isn't based on a transparent blockchain would help, or at least prevent the "easy" default choice (aka delisting)
[2019-09-25 12:10:36] <binaryFate> They have *zero* understanding, and the debate is often based on headlines and click bait type of statements. Changing that is a good objective.
[2019-09-25 12:10:46] <ArticMine> I see the actual FATF travel rule as a win or Monero vs chain analysis
[2019-09-25 12:11:01] <ajs[m]> Hi
[2019-09-25 12:11:13] <binaryFate> ArticMine I agree, but conveying that will be challenging
[2019-09-25 12:11:17] <midipoet> the answer should be - to protect against data mining, protect financial privacy, and adhere to GDPR regulations regarding protection of personal data
[2019-09-25 12:11:21] <binaryFate> hey ajs[m]
[2019-09-25 12:11:25] <arnuschky> From the regulation side we hear from lawyers that eg FinCEN is not at all opposed to privacy coins as they do see financial privacy as a must and are worried about the openess of transparent blockchains. However, this must be somehow married with AML/KYC laws to fly.
[2019-09-25 12:11:37] <xmrmatterbridge> <sgp_> I think the focus should be on helping people favorably interpret existing regulations
[2019-09-25 12:11:41] <krongle> @binaryFate - important thing is who are we trying to educate and explain to. Regulators? Politicians? Public at large? Not necessarily same strategy for the different groups. Especially the first two (regulators and politicians) will need to have something that is seen to be 'on their side'. Don't see just banging the fungibility drum as a
[2019-09-25 12:11:42] <krongle> workable strategy.
[2019-09-25 12:12:10] <ArticMine> The message needs to be privacy is nor incompatible with AML / KNC
[2019-09-25 12:12:20] <ArticMine> not
[2019-09-25 12:12:22] <binaryFate> IMO the outreach group is doing a great job at educating the public at large.
[2019-09-25 12:13:02] <almutasim> There are instances where the laws need to be changed, and we should be ready for that, too.
[2019-09-25 12:13:21] <almutasim> Outreach can coordinate with the efforts of this group.
[2019-09-25 12:13:23] <ArticMine> I do not see opposing AML/ KNC is the way to go
[2019-09-25 12:13:31] <arnuschky> What I am not seeing is a round table/group of operators (exchanges etc) that discuss how to deal with Monero in the current climate.
[2019-09-25 12:13:34] <xmrmatterbridge> <sgp_> Here is an example shared by the ECC which may help public opinion and reassure exchanges, even if it's not a legal opinion: https://electriccoin.co/blog/how-zcash-is-compliant-with-the-fatf-recommendations
[2019-09-25 12:14:41] <midipoet> ArticMine: if one is under AML investigation, pretty much all tools are available to investigators, afaiu. worst comes to worst, they ask for full access to wallet through proper legal procedures.
[2019-09-25 12:14:54] <binaryFate> Thanks sgp_. Producing more of that kind of content internally would surely help as right now each of us is re-writing / re-explaining from scratch everytime we interact with externals on this topic
[2019-09-25 12:15:09] <xmrmatterbridge> <sgp_> Asking people not to do AML isn't going to get any traction
[2019-09-25 12:15:25] <de_anna> what regulations stand to threaten the progress and adoption of monero? what regulations can be interpreted and adhered to or adaptable already? what new regulations and progressive proposals (i.e. KYC, AML etc.,) should be considered?
[2019-09-25 12:15:49] <nioc> hi, just catching up,  fluffy mentioned at konferenco that he is involved with getting lawyers to right up guidelines that will help exchanges to list monero
[2019-09-25 12:16:02] <binaryFate> indeed, getting a clearer list of what regulations are directly or indirectly impacting Monero (and vice-versa) would be a good start
[2019-09-25 12:16:14] <xmrmatterbridge> <sgp_> nioc: without going into the specifics, this is a slow and ongoing process
[2019-09-25 12:16:14] <almutasim> It would be good to know the status of those guidelines.
[2019-09-25 12:16:31] <ArticMine> The treat I see is a reliance on chain analysis for AML and its use as a replacement or KNC
[2019-09-25 12:16:40] <ArticMine> for
[2019-09-25 12:16:41] <nioc> sgp_: thx
[2019-09-25 12:16:45] <de_anna> exactly, a horizon scan and some mapping to start with
[2019-09-25 12:16:46] <binaryFate> nioc: I believe this is mostly about reassuging exchanges that XMR is not a security. Until a year ago they were crazy with that (due to ICOs and SEC crack down).
[2019-09-25 12:17:03] <binaryFate> I mean the past efforts with lawyers
[2019-09-25 12:17:14] <binaryFate> Ok we're touching a lot on the second point already.
[2019-09-25 12:17:29] <binaryFate> So, 2) what do you see happening in practice? How do we get to these objectives?
[2019-09-25 12:17:29] <xmrmatterbridge> <sgp_> binaryFate: no, it's mostly around other regulatory concerns. Monero is pretty clearly not a security
[2019-09-25 12:17:32] <midipoet> is the goal to ensure that VASPs can comply with regulations? or is the goal to ensure that regulations do not undermine one's ability to use Monero?
[2019-09-25 12:17:44] <de_anna> both?
[2019-09-25 12:17:47] <ArticMine> I agree with sgp
[2019-09-25 12:18:06] <de_anna> comply with the progressive ones at least? to start...
[2019-09-25 12:18:07] <ArticMine> on the security issue
[2019-09-25 12:18:33] <midipoet> because the former is a protocol discussion. the latter is a legislature discussion
[2019-09-25 12:18:50] <de_anna> and both are highly relevant imo
[2019-09-25 12:19:06] <de_anna> also where the capacity building and technical education comes in on both fronts
[2019-09-25 12:19:20] <xmrmatterbridge> <sgp_> I would like to see Zcash-type blog posts
[2019-09-25 12:19:33] <almutasim> That's a good idea.
[2019-09-25 12:19:35] <midipoet> yes, i agree - but the latter is the potentially the more important debate. it will be very difficult for us to change 5AMLD or the FATF guidelines
[2019-09-25 12:19:40] <xmrmatterbridge> <sgp_> Those are specific deliverables
[2019-09-25 12:20:00] <midipoet> however, we can potentially ensure the right to financial privacy through lobbying and affective political stragety
[2019-09-25 12:20:25] <binaryFate> I agree sgp_, more elaborated pieces of communication would go a long way. Could be reused in media articles etc.
[2019-09-25 12:20:25] <de_anna> precisely, and that leads into the domain expertise
[2019-09-25 12:20:37] <binaryFate> Often we are not offering any alternative to crappy narratives.
[2019-09-25 12:20:41] <arnuschky> I see two areas: 1) How/why Monero is treated different from other cryptocurrencies. 2) General AML/KYC in crypto.
[2019-09-25 12:21:05] <arnuschky> Not sure if we can do much about 2), but we should work on approaching 1) in order to keep Monero going imho
[2019-09-25 12:21:20] <xmrmatterbridge> <sgp_> afk for 5 mins
[2019-09-25 12:21:25] <arnuschky> Of course if 2) heavily counteracts what Monero stands for, we're in a bit of a bind
[2019-09-25 12:21:25] <de_anna> is there a need to go one level higher as well? the general arguments for digital privacy?
[2019-09-25 12:21:37] <ArticMine> Yes
[2019-09-25 12:21:37] <binaryFate> We're still very abstract. What can we do in practice?
[2019-09-25 12:21:47] <binaryFate> ^yes
[2019-09-25 12:22:10] <midipoet> 1) set of communication materials for journlaists/questions surrounding existing policy
[2019-09-25 12:22:30] <de_anna> depends on the categories, as midipoet just showed - lobbying if legislation, for example
[2019-09-25 12:22:52] <binaryFate> I would like to analyze and react to reports coming out like the latest one of the European Commission https://ec.europa.eu/jrc/en/publication/eur-scientific-and-technical-research-reports/blockchain-now-and-tomorrow
[2019-09-25 12:22:53] <midipoet> 2) attempt to form channels of communication into new policy (politicking)
[2019-09-25 12:23:09] <krongle> Hire a PR firm with specific outcomes? I think a lot of our communication is still targeted to people who think like us rather than trying to figure out how to convince people that don't think like us.
[2019-09-25 12:23:39] <binaryFate> krongle that's more the outreach group. They're our awesome PR firm imo :)
[2019-09-25 12:23:47] <almutasim> krongle That's a good point.
[2019-09-25 12:23:49] <midipoet> krongle: there are initiatives to get involved with policy discussions. the ITU, and ISO are two on the list to get involved with at more depth
[2019-09-25 12:24:08] <binaryFate> midipoet do you want to briefly talk about the ITU? As an example
[2019-09-25 12:24:11] <krongle> Yes - they are doing a great job, but I still don't think Joe public sees much of the monero outreach stuff. And if they do see it, they wouldn't understand it
[2019-09-25 12:24:35] <midipoet> the ITU announed an interesting digital finance group meeting in December. the ISO are currently engaging with standardisation. we should be involved with both
[2019-09-25 12:24:51] <almutasim> krongle We have been working on press releases to go broader.
[2019-09-25 12:24:53] <ArticMine> Publication of information on Monero and AML / KNC along the lines of what Zcash has domes above but more extensive and comprehensive
[2019-09-25 12:24:54] <de_anna> I have a rotating seat at the ISO
[2019-09-25 12:25:06] <midipoet> i sent a copy of the ITU invite to binaryFate, but others should get involved with ISO groups/national standards groups in their own country
[2019-09-25 12:25:12] <de_anna> also involved at the OECD level for disseminating policy proposals to nation states for blockchain and crypto
[2019-09-25 12:25:23] <de_anna> happy to use both as forums for dissemination and lobbying
[2019-09-25 12:25:28] <krongle> almutasim - great to hear re press releases
[2019-09-25 12:26:07] <midipoet> de_anna: that's great. but this group needs to decided on a coherent and consistent narrative
[2019-09-25 12:26:55] <ArticMine> Development of the relevant materials can be very supportive of the suggestions above
[2019-09-25 12:26:57] <almutasim> midipoet It seems we need some legal input to establish the narrative.
[2019-09-25 12:27:02] <midipoet> so we can all communicate the same thing. this needs work, as it combines social policy (financial privacy rights), economics (fungibility), and data protection (financial data)
[2019-09-25 12:27:10] <de_anna> agreed completely, until #1 above you suggested is done, and furthermore until there is consistency in the messaging we can't press go
[2019-09-25 12:27:13] <midipoet> almutasim - yes
[2019-09-25 12:27:41] <binaryFate> Ok, so far I hear mostly writing of reports / blog posts about Monero & regulations. And sneaking ourselves in relevant industry discussions or at least follow what they do and how it could impact the project.
[2019-09-25 12:27:57] <binaryFate> The former supporting the later indeed.
[2019-09-25 12:28:22] <almutasim> If fluffypony mentioned ongoing work by attorneys to put together guidelines (as was commented above), it might be nice to tie into that to set the narrative.
[2019-09-25 12:28:36] <midipoet> yes, and also this is where i can plug privacyseries.org - which is an initiative to try and get 21 people in a room to talk through these issues.
[2019-09-25 12:28:56] <de_anna> The best thing one could go into any of these forums with is concrete arguments and coherent/accessible technical interpretations of regulations in relation to midipoet's three areas above
[2019-09-25 12:28:56] <midipoet> 7 cc devs. 7 regulators. 7 academics.
[2019-09-25 12:29:04] <binaryFate> midipoet bring it on
[2019-09-25 12:29:15] <ArticMine> almutasim That needs to be part of the material support
[2019-09-25 12:29:29] <arnuschky> Can we set up an industry discussion group? I mean it's probably most productive when we get the people operating monero-based service together, as they are the ones being directly impacted.
[2019-09-25 12:29:56] <de_anna> often industry is the strongest lobby as well
[2019-09-25 12:30:03] <almutasim> arnuschky Good idea.
[2019-09-25 12:30:06] <de_anna> that being said, I feel like it wouldn't be the largest industry group in scope...
[2019-09-25 12:30:23] <binaryFate> Yeah that's probably why it does not exist yet :)
[2019-09-25 12:30:45] <binaryFate> Would be great if something like that was setup yes.
[2019-09-25 12:31:11] <binaryFate> midipoet so what about privacyseries.org? Purpose?
[2019-09-25 12:31:16] <arnuschky> de_anna: I don't understand what you mean.
[2019-09-25 12:31:22] <midipoet> does anybody have good contact with the Kraken guy who spoke at DefCon?
[2019-09-25 12:31:26] <arnuschky> I think we'd happily join a groupd like that.
[2019-09-25 12:31:48] <midipoet> binaryFate - basically to try and get people talking from different stakeholder groups
[2019-09-25 12:32:00] <midipoet> who wouldnt neccesarily talk otherwise
[2019-09-25 12:32:16] <midipoet> as in reality, humanity is trying to figure this out, but often it seems in isolation
[2019-09-25 12:32:17] <binaryFate> what sort of projects do they have onboard? Or is it still preliminary?
[2019-09-25 12:32:18] <de_anna> arnuschky I think there's definitely potential, it will be a small industry group at first but there's nothing wrong with that so long as it's effective - I also think it would create the conditions for more to join and/or enter the space
[2019-09-25 12:32:28] <krongle> I also think such a group could have a voice of some authority, even if small. Try and leverage some of the negative attention we get by getting our "leading privacy coin think-tank" quoted as an alternative viewpoint every time someone says something bad about us.
[2019-09-25 12:32:32] ⇐ msvb-lab quit (~michael@x5f76a4bd.dyn.telefonica.de): Ping timeout: 245 seconds
[2019-09-25 12:32:58] <midipoet> no. it should be financial privacy think tank
[2019-09-25 12:33:13] <midipoet> i think its a larger debate
[2019-09-25 12:33:16] <binaryFate> Pipe dream: organize a privacy-oriented conference to bring together projects and domains that otherwise would not talk to each other and realize they are fighting the same fight
[2019-09-25 12:33:26] <midipoet> binaryFate - yes
[2019-09-25 12:33:32] <de_anna> <3
[2019-09-25 12:33:34] <ArticMine> Great idea
[2019-09-25 12:33:35] <de_anna> yep
[2019-09-25 12:33:39] <midipoet> or at least so that different groups can talk about how they see it
[2019-09-25 12:33:44] <krongle> Would be effective is such a group can make active statements that are ahead of the curve - e.g. this is what financial privacy could look like under FATF. If you are the first to say something, you don't have to say anything that ground-breaking in order to get publicised.
[2019-09-25 12:33:46] <arnuschky> I think there's the larger debate (financial privacy think tank), but I fear that this might be very abstract and broad
[2019-09-25 12:33:47] <de_anna> and lead them, or at least provide alternatives
[2019-09-25 12:34:26] <de_anna> midipoet is Privacy Series 2020 oriented towards being a privacy think tank or?
[2019-09-25 12:34:39] → msvb-lab joined (~michael@x4d095adb.dyn.telefonica.de)
[2019-09-25 12:34:43] <midipoet> de_anna: a financial privacy meeting
[2019-09-25 12:34:44] <arnuschky> and the way more practical questions of operating a Monero-based service ("my regulator requires me to use chainalysis - do I have to delist Monero?")
[2019-09-25 12:34:54] <midipoet> to flesh out what exactly should financial privacy look like
[2019-09-25 12:35:01] <midipoet> if it should exist at all
[2019-09-25 12:35:13] <midipoet> and should include all sides (as some will think it shouldnt exist)
[2019-09-25 12:35:24] <de_anna> relevant and solid approach
[2019-09-25 12:35:27] <midipoet> and that is fair game - but at least sitting together to talk about it will help
[2019-09-25 12:35:43] <ArticMine> ("my regulator requires me to use chainalysis - do I have to delist Monero?") <--- Can o provide us with a copy of the relevant laws / regulations
[2019-09-25 12:35:51] <midipoet> KU Leaven in Belgium (Claudia Diaz) has said they will host
[2019-09-25 12:35:57] <midipoet> she is a cryptography and anonymity expert
[2019-09-25 12:36:37] <midipoet> i have not set a date, as it means commiting, and i wished to see what konferenco was doing (timing wise)
[2019-09-25 12:36:44] <midipoet> as timing them together would be interesting
[2019-09-25 12:37:00] <midipoet> https://homes.esat.kuleuven.be/~cdiaz/
[2019-09-25 12:37:01] <binaryFate> I knew that name ringed a bell: https://www.monerotalk.live/live-w-peter-todd-and-claudia-diaz-nym-at-the-mcc2019
[2019-09-25 12:37:07] <midipoet> yes
[2019-09-25 12:37:10] <midipoet> she was at MCC
[2019-09-25 12:37:46] <binaryFate> Ok I think we are touching on the last point to define the scope. Should it be Monero specific? Cryptocurrency agnostic? Privacy oriented?
[2019-09-25 12:38:07] <midipoet> my view is "financial privacy"
[2019-09-25 12:38:13] <xmrmatterbridge> <sgp_> To some extent it needs to be Monero-specific since Monero is the only one that has mandatory privacy
[2019-09-25 12:38:32] <xmrmatterbridge> <sgp_> It's the only one that can actually be called "fungible" with strong backing arguments
[2019-09-25 12:38:33] <midipoet> lol
[2019-09-25 12:38:38] <almutasim> Monero centric.
[2019-09-25 12:39:03] <de_anna> financial privacy, the minimum technical parameters for achieving this, any cryptocurrencies that meet these standards (selective financial privacy oriented agnosticism)
[2019-09-25 12:39:07] <ajs[m]> 3) support research and publication of peer reviewed articles in tier 1 journals.
[2019-09-25 12:39:12] <xmrmatterbridge> <sgp_> Zcash compliance is effectively the same as Bitcoin compliance since it's all transparent
[2019-09-25 12:39:26] <midipoet> ajs[m]: yes. but difficult!
[2019-09-25 12:39:44] <midipoet> especially the tier 1 (4 star) bit!
[2019-09-25 12:39:54] <ArticMine> I would say Monero centric but not exclusive. Think Konfernco / Monero Village
[2019-09-25 12:39:54] <binaryFate> I could summarize my view by saying it should be similar to the successful position of the Monero Research Lab. Clear Monero branding, but a lot of interaction with other scholars, and study broad topics that _eventually_ are relevant to Monero (not necessarily right away).
[2019-09-25 12:40:44] <midipoet> there is also the question of bias and neutrality. if we remove it from Monero, then we are just interested stakeholder - as apposed to the main lobbyists
[2019-09-25 12:40:48] <Inge-> I'm ready for an article like "Privacy coin Monero compatible with FATF Travel Rule"
[2019-09-25 12:41:09] <de_anna> in many respects Monero is the only one (or one of few depending on your bar) that adheres to this anyway, so it would naturally gravitate towards it
[2019-09-25 12:41:12] <midipoet> Inge-: we need a lawyer to state that though
[2019-09-25 12:41:21] <xmrmatterbridge> <sgp_> "Monero is the most FATF-compatible cryptocurrency"
[2019-09-25 12:41:22] <xmrmatterbridge> <sgp_> "Monero is the most GDPR-compatible cryptocurrency"
[2019-09-25 12:41:29] <xmrmatterbridge> <sgp_> This is low-hanging fruit people
[2019-09-25 12:41:36] <midipoet> yes to GDPR debate
[2019-09-25 12:41:54] <de_anna> agreed and bringing that back to the corporate/industry positioning is also key
[2019-09-25 12:42:07] <ArticMine> I would go much further. "Privacy coin Monero very useful with FATF Travel Rule compliance"
[2019-09-25 12:42:08] ⇐ liberiga quit (~liberiga@gateway/tor-sasl/liberiga): Ping timeout: 260 seconds
[2019-09-25 12:42:09] <binaryFate> I still want to specs GDPR-coin and show it's Monero ^^
[2019-09-25 12:42:29] <midipoet> binaryFate: i have a meeting with University of Graz regarding this
[2019-09-25 12:42:38] <midipoet> they wrote the recent Bitcoin and GDPR compliance paper
[2019-09-25 12:42:49] <midipoet> i wil share now for those that have not already seen it
[2019-09-25 12:43:00] <binaryFate> Everyone seems pretty aligned about scope. Monero branded, Monero oriented, but not exclusive
[2019-09-25 12:43:07] <xmrmatterbridge> <sgp_> yes
[2019-09-25 12:43:11] <binaryFate> Now what about a name?
[2019-09-25 12:43:15] <midipoet> https://usercontent.irccloud-cdn.com/file/4Plwnw4T/Buocz-Bitcoin_and_the_GDPR.pdf
[2019-09-25 12:43:52] <binaryFate> I sort of assume it will be a group of its own, sorry if I skip one step. My impression from reddit discussions was there was reasonable consensus for that
[2019-09-25 12:44:35] <ArticMine> binaryFate Monero compliance with the GDPR is huge.
[2019-09-25 12:44:53] <binaryFate> Ikr, we could play that so well
[2019-09-25 12:44:59] <midipoet> yes i agree
[2019-09-25 12:45:06] <de_anna> is there agreement that this is a think tank? or a working group?
[2019-09-25 12:45:24] <midipoet> working-group sounds more action orientated
[2019-09-25 12:45:29] <almutasim> Agreed.
[2019-09-25 12:45:39] <de_anna> agreed unless you go with the 'think and do tank' ;)
[2019-09-25 12:45:52] <binaryFate> I like think tank for the weight it conveys though. Remember we want to be quoted everywhere and participate in meetings etc.
[2019-09-25 12:45:53] <midipoet> otherwise we could just sit around philosophising all the time
[2019-09-25 12:46:13] <de_anna> Think tank with sub working groups according to domain expertise and objectives?
[2019-09-25 12:46:20] <binaryFate> midipoet not if we write reports as we said. This is just about getting an effective branding.
[2019-09-25 12:46:30] <midipoet> ah i see
[2019-09-25 12:46:47] <binaryFate> so not rediscussing what we do, just how we can present ourselves and our material
[2019-09-25 12:46:58] <midipoet> FPTT
[2019-09-25 12:47:08] <binaryFate> "Monero regulatory think tank"
[2019-09-25 12:47:15] <de_anna> a think tank carries weight, it is easier as a structure to obtain funding in as well
[2019-09-25 12:47:21] <midipoet> i thought we were staying agnostic?
[2019-09-25 12:47:37] <almutasim> Does "think tank" carry weight?
[2019-09-25 12:47:46] <de_anna> haha fair enough
[2019-09-25 12:47:53] <ArticMine> I suggest  Monero regulatory and compliance workgroup
[2019-09-25 12:48:17] <binaryFate> ArticMine what's your reasoning to prefer workgroup over think tank?
[2019-09-25 12:48:26] <almutasim> The word compliance is a bit of a downer.
[2019-09-25 12:48:31] <midipoet> RCTT
[2019-09-25 12:48:33] <binaryFate> I also like think tank because it's much easier to invite externals to contribute on a specific report for instance
[2019-09-25 12:48:37] <ArticMine> Action  / support oriented
[2019-09-25 12:48:37] <krongle> I think it should be more than just presentation (that's outreach). I think it should produce and propose practical solutions to problems faced by monero community and monero service providers in the face of current legislation
[2019-09-25 12:48:38] <binaryFate> Rather than "join our workgroup"
[2019-09-25 12:48:42] <midipoet> regulatory and compliance think tank
[2019-09-25 12:48:56] <arnuschky> I like think tank to span the whole thinkg. We can then have an "exchange compliance workgroup" or "FATF travel rule work group"
[2019-09-25 12:48:56] <xmrmatterbridge> <sgp_> Do you have any entities that are willing to join?
[2019-09-25 12:48:57] <midipoet> almutasim: lol
[2019-09-25 12:49:19] <de_anna> i think compliance shows a progressive and inclusive positioning as well, even though the regulations need an overhaul
[2019-09-25 12:49:24] <ArticMine> Part of the purpose is to support VASPs with compliance
[2019-09-25 12:49:38] <midipoet> yes, and GDPR compliance is a narrative that we should/will push
[2019-09-25 12:49:40] <xmrmatterbridge> <sgp_> I think compliance is a larger part than the regulatory side
[2019-09-25 12:49:42] <almutasim> We can support compliance without having it in the name.
[2019-09-25 12:49:42] <binaryFate> ArticMine we can be action / support oriented in any case, by producing outcomes to do so, regardless of branding
[2019-09-25 12:49:46] <midipoet> and that is a very powerful one
[2019-09-25 12:50:47] <de_anna> having it in the name would be a matter of signalling, if that is the intent of the community
[2019-09-25 12:50:56] <binaryFate> I am keen on establishing lasting connections with externals, and working group is a much harder sell to get them to gravitate around us (it sounds much commital)
[2019-09-25 12:51:30] <ArticMine> I am not opposed to think tanks it st sounds academic and cerebral to me
[2019-09-25 12:51:34] <xmrmatterbridge> <Kayla> hoofty? u there? 😮
[2019-09-25 12:51:34] <almutasim> I like think tank better than compliance.
[2019-09-25 12:51:36] <midipoet> i agree with that binaryFate
[2019-09-25 12:52:11] <de_anna> and arnuschky gave a good working example of that above ^
[2019-09-25 12:52:27] <binaryFate> Anyway, we don't need to decide on name now if there is no clear winning proposition.
[2019-09-25 12:52:38] <binaryFate> Which brings me to the last point. How do we go forward?
[2019-09-25 12:52:41] <de_anna> structure is important, however
[2019-09-25 12:52:48] <binaryFate> Practically, how do we organize?
[2019-09-25 12:52:54] <hooftly> Hey Kayla
[2019-09-25 12:52:55] <krongle> 'think tank' is fine - but we need to have something else in the title. Can't just have a general tank that thinks about stuff :-)
[2019-09-25 12:53:19] <almutasim> krongle Ha.
[2019-09-25 12:53:23] <midipoet> maybe a Taiga?
[2019-09-25 12:53:24] <binaryFate> I think this was a good meeting and another could be useful but what is everyone opinion?
[2019-09-25 12:53:26] <ArticMine> Also while GDPR is the obvious place to start the scope needs to be broadened to privacy and consumer data legislation in general
[2019-09-25 12:53:33] <almutasim> We need to connect with lawyers.
[2019-09-25 12:53:39] <xmrmatterbridge> <Kayla> OH SHIT IT WORKED 😮
[2019-09-25 12:53:46] <almutasim> This was a great kickoff meeting!
[2019-09-25 12:53:47] <midipoet> ArticMine - just GDPR and Blockchain
[2019-09-25 12:54:00] <midipoet> GDPR does not care what the data is
[2019-09-25 12:54:17] <midipoet> unless its "sensitive data"
[2019-09-25 12:54:25] <binaryFate> Taiga sounds good, it worked for other Monero initiatives at least
[2019-09-25 12:54:29] <ArticMine> I mean privacy legislation in other countries
[2019-09-25 12:55:02] <almutasim> Taiga is good.
[2019-09-25 12:55:05] <midipoet> ArticMine: yes, this is true. however we can use compliance in one jusrisdiction as leverage
[2019-09-25 12:55:14] <de_anna> should we work towards a collective proposal outlining a think tank, it's purpose and structure, clear working groups on the above subjects, open it up and allow for further iteration and comments?
[2019-09-25 12:55:39] <ArticMine> I do agree tat the EU GDPR should be the start
[2019-09-25 12:56:10] <midipoet> yes, especially as other jurisdictions are looking to GDPR as example
[2019-09-25 12:56:12] <de_anna> after that solicit those that want to lead the think tank, working groups and any other functions required?
[2019-09-25 12:56:57] <binaryFate> I like the idea of a manifesto (grandiose word for the think tank defintion / proposal) to kick things of
[2019-09-25 12:57:07] ⇐ kico quit (~kico@gateway/tor-sasl/kico): Remote host closed the connection
[2019-09-25 12:57:09] <arnuschky> yes sounds good de_anna
[2019-09-25 12:57:11] → kic0 joined (~kico@gateway/tor-sasl/kico)
[2019-09-25 12:57:13] <binaryFate> Especially at the beginning, it's important that we get traction to get rolling
[2019-09-25 12:57:21] <de_anna> binaryFate I love me some manifestos, bring it
[2019-09-25 12:57:37] <xmrmatterbridge> <sgp_> I work for DV Chain (largest Monero OTC desk) and if other entities are interested in joining, I can see if we would like to as well. Of course that's not up to me though and I'm not promising anything
[2019-09-25 12:57:37] <krongle> Don't necessarily agree that GDPR is the place to start. That's us patting ourselves on the back, and not where the heat is coming from. More useful is to address KYC type legislation, which is what is most at odds (currently) with monero community.
[2019-09-25 12:58:00] <midipoet> krongle. yes, but GDPR is proactive
[2019-09-25 12:58:08] <midipoet> KYC.AML is reactionary
[2019-09-25 12:58:09] <de_anna> a select group (us in here now + those that missed out and want to form this) for the manifesto and proposal V1 and then open it up to the community at large
[2019-09-25 12:58:30] <binaryFate> Ok, those interested can draft a manifesto and we iterate. Go go manifestos.
[2019-09-25 12:58:37] <xmrmatterbridge> <sgp_> I also don't expect GDPR to take up months of time either. Maybe a week? Run a victory lap and get that writeup out in the open
[2019-09-25 12:58:40] <almutasim> de_anna Sounds good.
[2019-09-25 12:58:45] <binaryFate> Shall we have a continuous discussion channel? IRC? Wire?
[2019-09-25 12:58:46] <krongle> I am part of xmr.to, and we would definitely be interested in joining. As a matter of practical relevance to us.
[2019-09-25 12:58:51] <midipoet> sgp_: oh i dont know about that
[2019-09-25 12:58:55] <binaryFate> I prefer IRC personally.
[2019-09-25 12:59:06] <midipoet> GDPR is fairly hefty
[2019-09-25 12:59:27] <ArticMine> It is also very related. For example using chain analysis for AML and to replace KNC creates a direct conflict with the GDPR
[2019-09-25 12:59:57] <de_anna> IRC and I think it should be broken down into sections and goals i.e. manifesto framework, discussion on working-groups (which is definitely where people have a lot of opinions and more might want to join) etc.,
[2019-09-25 13:00:01] <arnuschky> sgp_: we would be interested in that too
[2019-09-25 13:00:13] <midipoet> ArticMine: yes. all psuedo-anonymised data is subject to GDPR
[2019-09-25 13:00:31] <krongle> I think the sad truth these days is that the default assumption from the regulators is that you are supposed to do KYC _ and _ chain analysis. Our job to propose / justify alternatives
[2019-09-25 13:00:41] <midipoet> and the secondary processing of personal data (even if pseudoanonymised) is subject to GDPR compliance
[2019-09-25 13:00:49] <binaryFate> I suggest another meeting in a week (later does not need to be as frequent but if we want to get moving there is much to define and do)
[2019-09-25 13:01:07] <ArticMine> Same time / place?
[2019-09-25 13:01:16] <xmrmatterbridge> <sgp_> I'm a huge proponent of keeping the scope small and getting content out quickly fwiw
[2019-09-25 13:01:16] <almutasim> Sounds good.
[2019-09-25 13:01:17] <midipoet> will there be a Taiga in the meantime?
[2019-09-25 13:01:27] <de_anna> agreed, and perhaps half dedicated to higher level objectives of the manifesto - the second on a discussion to narrow the focus to perhaps 3 working-groups to start? for ex
[2019-09-25 13:01:37] <xmrmatterbridge> <sgp_> The Zcash blog posts can be written in a day, for example
[2019-09-25 13:01:51] <midipoet> yes, but i would not take ZCAsh methods as anything
[2019-09-25 13:01:59] <midipoet> they have proven their incompetence on a number of levels, imo
[2019-09-25 13:02:19] <midipoet> showing screen grabs as regulatory compliance is nonsense, imo
[2019-09-25 13:02:20] <xmrmatterbridge> <sgp_> They have a seat at the table, so even if you think they're incompetent, they're approaching regulators better than we are right now
[2019-09-25 13:02:29] <midipoet> what table?
[2019-09-25 13:02:37] <binaryFate> In the meantime, we can join #monero-compliance and discuss stuff like manifest / setting up taiga etc
[2019-09-25 13:02:44] <xmrmatterbridge> <sgp_> they spoke at the FATF hearings iirc
[2019-09-25 13:02:48] <midipoet> they bought their influence , as far as i can see it
[2019-09-25 13:02:52] <binaryFate> name is temporary, just so discussions don't get lost in community channel over the coming week
[2019-09-25 13:02:58] <midipoet> due to links with JP Morgan
[2019-09-25 13:03:03] <arnuschky> Who will be anyone at HCPP? That is next weekend.
[2019-09-25 13:03:03] <midipoet> and got absolutely nowhere
[2019-09-25 13:03:18] <de_anna> arnuschky I will!
[2019-09-25 13:03:23] <binaryFate> me too
[2019-09-25 13:03:28] <xmrmatterbridge> <sgp_> midipoet: you seem to be holding a bigger grudge than necessary here. I'm just looking at what they have done that's effective as inspiration. Let's start there and make it better
[2019-09-25 13:03:34] <de_anna> late night whiskey monero-compliance session anyone?
[2019-09-25 13:04:03] <midipoet> sgp_: perhaps. all i am saying is that screen grabs of a proposed wallet feature is not regulatory compliance
[2019-09-25 13:04:17] <binaryFate> There will be quite some Monero people at HCPP. We can discuss this group, and also CCC preparation there.
[2019-09-25 13:04:23] <xmrmatterbridge> <sgp_> Last time we tried this we probably bit off more than we could chew, so I want to make sure we have actionable goals that will be completed
[2019-09-25 13:04:35] <de_anna> midipoet and xmrmatterbridge this is exactly the discussions to be had when we clarify the working groups and their respective purposes
[2019-09-25 13:04:37] <binaryFate> We're over time. Thank you everyone for the meeting! (you can stay and continue)
```

**Comment by erciccione** [comment]
@SamsungGalaxyPlayer could you post the logs on repo.getmonero.org? All other meeting logs are posted there, would be nice to have this one as well.

**Comment by SamsungGalaxyPlayer** [comment]
This issue should be closed after the meeting logs are posted to the website

**Comment by sanecito** [comment]
(Closed as the meeting happened over a year ago at this point)
